src/store.ts

```diff
@@ -13,13 +13,33 @@
 
 export interface JwtStore {
```

Comment:
should this just be renamed to Store since as i understand it, some of these methods have nothing to do with JWTs?

Reply:
@tackley had the same idea!

graham - what's a good new name?

Reply:
after cycling round on this, we've decided to keep the store of recordings separate to the jwt one. They're quite different things.